src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -522,6 +522,11 @@ public Long getFsGroup() {
 	 */
 	private PodAntiAffinity podAntiAffinity;
 
+	/**
+	 * A custom init container image name to use when creating a StatefulSet
+	 */
+	private String statefulSetInitContainerImageName;
```

Comment:
Shouldn't this default to "busybox" as well?

Reply:
the intent is to only set it if you want to specify a custom image name. it could be defaulted here instead tho, i can go either way..